packages/app/features/settings/AppInfoScreen.tsx

```diff
@@ -75,6 +147,34 @@ export function AppInfoScreen(props: Props) {
             </View>
           )}
 
+          <XStack
+            key="debug-toggle"
+            justifyContent="space-between"
+            alignItems="center"
+            padding="$l"
+          >
+            <SizableText flexShrink={1}>Enable Developer Logs</SizableText>
+            <Switch
+              style={{ flexShrink: 0 }}
+              value={enabled}
+              onValueChange={toggleDebugFlag}
+            ></Switch>
+          </XStack>
+
+          {enabled && logs.length > 0 && (
+            <Stack>
+              <Button onPress={onUploadLogs}>
+                <Text>Upload logs ({logs.length})</Text>
+              </Button>
+            </Stack>
+          )}
+          {enabled && logId && !hasClients && (
+            <YStack padding="$l">
+              <Text>Please email support@tlon.io with this log ID:</Text>
```

Comment:
out of scope, but would be cool to dm Tlon Team with the ID

Reply:
yeah that's not a bad idea